### DISCUSSION TITLE
`displayedAttribtues` behavior with the document routes

### DISCUSSION THREAD

**Discussion by curquiza** [meilisearch/product#432] [Feedback & Feature Proposal]
Currently, in the search engine (v0.27.0), the displayed attributes change the display of two kinds of routes
- the search route: `GET/POST indexes/{uid}/search`
- the document routes: `GET indexes/{uid}/documents` and `GET indexes/{uid}/documents/{doc_uid}`

Knowing the other settings are only related to the search, shouldn't we only restrict the `displayedAttributes` to the search routes?

⚠️ In the future, the `filterableAttributes` and the `sortableAttributes` might also concern the document routes, for example to delete a specific batch of documents (where the price is greater than ...)

**Comment by curquiza** [comment]
I discussed with @gmourier that only applying `displayedAttributes` to `/search` will let a possibility for the users to see their raw documents without any setting applied. Indeed, `/documents` is a management route and not an end-user route.

**Comment by gmourier** [comment]
Following the [v0.28 release](https://github.com/meilisearch/meilisearch/releases/tag/v0.28.0), this discussion is now locked.

We encourage you to create a new thread if needed.

Thank you ✨
